scalding-serialization/src/main/scala/com/twitter/scalding/serialization/macros/ExportMacros.scala

```diff
@@ -0,0 +1,24 @@
+package com.twitter.scalding.serialization.macros
+
+import com.twitter.scalding.serialization.Exported
+import com.twitter.scalding.serialization.{OrderedSerialization, DefaultOrderedSerialization}
+import scala.reflect.macros.whitebox
+
+class ExportMacros(val c: whitebox.Context) {
```

Comment:
are we actually calling this anywhere?

Reply:
Nope, i think its killable/not sure if adds something. Was discussing that with @travisbrown towards EOD here.

Reply:
Right now the only exported instance has a concrete type, but once you have generically derived ones the macro here will be necessary (although it turns out it doesn't need to be whitebox, I believe). I've got some notes on this from when I was prepping [this talk](https://meta.plasm.us/slides/scalaworld/)—I'll try to find them.